doc/user-manual/language/universe-levels.lagda.rst

```diff
@@ -300,8 +300,8 @@ to the right sort once their arguments are known.
 
 More examples:
 
-* ``piSort Level (\l. Set l)`` evaluates to ``Setω``
-* ``piSort (Set l) (\_. Set l')`` evaluates to ``Set (l ⊔ l')``
+* ``piSort Level (λ → Set l)`` evaluates to ``Setω``
+* ``piSort (Set l) (λ → Set l')`` evaluates to ``Set (l ⊔ l')``
```

Comment:
Looks like we are missing a bound `l` in the first case and an underscore in the second.

Reply:
Thanks!